test/scripts/streaming/run-all-tests.sh

```
@@ -0,0 +1,61 @@
+echo "Testing against: $TARGET_URL"
+echo
+
+# Test 1: Streaming Detection (quick mode)
+echo "1. Running streaming detection test (quick mode)..."
+python test-streaming-detection.py --url "$TARGET_URL" --quick || {
+    echo "❌ Streaming detection test failed"
+    exit 1
+}
+echo
+
+# Test 2: Streaming Latency (3 questions)
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)